x/cronos/keeper/precompiles/relayer.go

```diff
@@ -11,8 +11,7 @@ import (
 	ibckeeper "github.com/cosmos/ibc-go/v7/modules/core/keeper"
 )
 
-// TODO adjust the gas cost
-const RelayerContractRequiredGas = 10000
+const RelayerContractRequiredGas = 300000
```

Comment:
https://cronos.yummy.capital/accounts/crc1tszdy28w2gxldpp0ww6ry58vqhy6rhmx7v6yf6

On average, two ibc msgs would used ~600,000 gas. (Update client is the most gas consuming operation with ard 400,000 gas)

Reply:
Following the above discussions, If we are to hard-coded a static value, I don't think we should take an average and apply to all message types. We should use a value that is always high enough to cover the cost, even if it means we may over-charge the user, as a temporary solution until we change to the Cosmos gas meter.

Reply:
Do you mean the max of all msg?

I think there is a huge difference between `update_client` and `rev_packet`. Could be better to separate in two differents pre compiles

Reply:
We can have a map, and check the prefix of user input

Reply:
> Do you mean the max of all msg?

Not a single max for all msg, sorry for the confusion. I am expecting each msg has its own max.